docs/UsersGuide/source/Quickstart.rst

```diff
@@ -66,16 +66,16 @@ For a detailed explanation of how to build and run the SRW App on any supported
             cmake .. -DCMAKE_INSTALL_PREFIX=..
             make -j 4  >& build.out &
 
-   #. Download and stage data (both the fix files and the :term:`IC/LBC` files) according to the instructions in :numref:`Section %s <DownloadingStagingInput>` (if on a Level 2-4 system).
+   #. Download and stage data (both the fix files and the :term:`IC/LBC <IC/LBCs>` files) according to the instructions in :numref:`Section %s <DownloadingStagingInput>` (if on a Level 2-4 system).
 
    #. Configure the experiment parameters.
 
       .. code-block:: console
 
-         cd regional_workflow/ush
-         cp config.community.sh config.sh
+         cd ush
+         cp config.community.yaml config.yaml
```

Comment:
It maybe good to mention that if the user has a shell script config file, he can either do any of the following

a) or use it as is by changing the following parameter in `config_defaults.yaml`
```
EXPT_CONFIG_FN: "config.sh"
```
b) convert it to yaml with
```
./config_utils.py -c $PWD/config.sh -t $PWD/config_defaults.yaml -o yaml >config.yaml
```

Reply:
@danielabdi-noaa I think this is a good thing to mention, but I will probably wait to add it to the main Build/Run chapter when I make those updates. The Quickstart chapter is supposed to be as succinct as possible, so I just show one way to run the workflow and refer the users to other chapters for additional options.

Reply:
Sounds good! Just one more thing I forgot to mention: The user can check a validity of a given config file by comparing it against `config_defaults.yaml`. For example
```
$ ./config_utils.py -c $PWD/config.yaml -v $PWD/config_defaults.yaml 
SUCCESS
```
If there is an invalid entry, you will get something lik
```
$ ./config_utils.py -c $PWD/config.yaml -v $PWD/config_defaults.yaml
INVALID ENTRY: ACCPP_PHYS_SUITE=FV3_GFS_v15p2
FAILURE
```